config/main.py

```diff
@@ -1155,6 +1155,22 @@ def validate_gre_type(ctx, _, value):
     except ValueError:
         raise click.UsageError("{} is not a valid GRE type".format(value))
 
+def multi_asic_save_config(db, filename):
+    """A function to save all asic's config to single file
+    """
+    all_files_config = {}
+    cfgdb_clients = db.cfgdb_clients
+
+    for ns, config_db in cfgdb_clients.items():
+        current_config = config_db.get_config()
+        sonic_cfggen.FormatConverter.to_serialized(current_config)
+        asic_name = "localhost" if ns == DEFAULT_NAMESPACE else ns
+        all_files_config[asic_name] = sort_dict(current_config)
+    click.echo("Integrate each ASIC's config into a single JSON file {}.".format(filename))
```

Comment:
Hi @judyjoseph , It is instructional message. 
It will not cause any sonic-mgmt test failure, because we didn't allow multiasic to save to single file without this PR implementation.
It is an extention for config save usage when multiasic and specify one file to save all asic config.